tox.ini

```diff
@@ -35,7 +35,7 @@ commands =
     # isolated from other tests
     # NOTE 2: We write junit XML file for nicer visualization on Circle CI
     rm -rf test-results
-    py.test scalyr_agent/tests scalyr_agent/builtin_monitors/tests -vv --durations=5 -m "not memory_leak and not json_lib" --junitxml=test-results/junit-1.xml
+    py.test scalyr_agent/tests scalyr_agent/builtin_monitors/tests scalyr_agent/monitor_utils/tests -vv --durations=5 -m "not memory_leak and not json_lib" --junitxml=test-results/junit-1.xml
```

Comment:
Good catch.

In the future we could perhaps also re-organize this a bit more to utilize py.test markers for everything and only specify top-level directory for tests.

Right now some of the tests intentionally run as part of a separate process to avoid cross test pollution.

Reply:
Yeah, good idea.

Reply:
We could also make a sane directory structure for tests.  I was going to suggest we check test inclusion using a pylint checker, but since this is an `ini` file, it makes importing some sort of shared "these are the test directories" variable a little more difficult (I believe).

Reply:
I agree.

@ArthurKamalov and I already briefly talked about that in the past.

I personally think all the tests should either live in ``tests/`` or ``scalyr_agent/tests`` directory (and sub-directories, as necessary).